### PR TITLE
[release] Bumped 2022.3.0

### PR DESCRIPTION
**Pull request by italovalcy**
Closes N/A

### Summary

- Bumped 2022.3.0

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -6,6 +6,22 @@ All notable changes to the MEF_ELine NApp will be documented in this file.
...
+=======
+
+Removed
+=======
+
+Fixed
+=====
+
+
+[2022.3.0] - 2022-01-23
```
Comment (rmotitsuki) [inline CHANGELOG.rst:22]: 2023-01-23

Comment (italovalcy) [reply CHANGELOG.rst:22]: Good catch, @rmotitsuki ! Thank you!!

Comment (italovalcy) [inline CHANGELOG.rst:22]:
```suggestion
[2022.3.0] - 2023-01-23
```

Comment (italovalcy) [reply CHANGELOG.rst]: Addressed in commit 572ec72

**Comment by italovalcy** [comment]
> missing setup.py property NAPP_VERSION = '2022.3.0'

Thanks for catching this, Rogério! I've pushed commit 388c6a8 to fix this.

**Review by rmotitsuki — APPROVED** [review]

**Pull request merged**